docs/faq/frequently_asked_questions.rst

```diff
@@ -0,0 +1,56 @@
+    * For all modules, you will need a UART-to-Serial adapter in order to communicate with your module using your computer. We recommend using a `CP2102 FTDI adapter <https://www.amazon.com/Ximimark-Module-Serial-Converter-CP2102/dp/B07T1XR9FT?>`_.
+
+#. Why is my module slowing down, even though I am not changing the throttle or increasing the throttle? Why is my module derating?
+    * Each motor has built in safeties to protect the module from overheating failures. The module will start to spin slower, or derate, when the safeties are in effect to keep it at a safe operating speed. To learn more about these safety features, please refer to the :ref:`Safety Systems <manual_safety>` documentation.
+    * Each module has a max-continuous thrust region depending on the size of the propeller you are using. Please refer to your module’s `Thrust Data <https://www.vertiq.co/thrust-data>`_ to learn more about its max-contintinous thrust.
+    * Each module also has a continuous torque limit, which you can find on your module’s datasheet on our `website <https://www.vertiq.co/>`_ by navigating to your module’s family page under the Documentation section. This datasheet can also help you choose the correct propeller size for your module
+
+#. How do I update my module’s firmware?
+    * You can find the latest firmware for your module on our `website <https://www.vertiq.co/>`_ by navigating to your module’s family page under the Firmware section. Please refer to our :ref:`Updating Firmware with Control Center <updating_firmware_control_center>` documentation on how to flash new firmware onto your module using Control Center.
+
+#. Why are there no throttle percentages on your thrust data?
+    * We do not include a column for throttle percentages in our thrust data because our modules support multiple throttle modes (PWM, Voltage, and Velocity), which means “throttle” can be interpreted differently between each mode. For example, a 75% throttle command in PWM mode can be interpreted as “Command 75% of the supplied voltage”. On the other hand, a 75% throttle command in Velocity mode can be interpreted as “Command 75% of the max velocity”, which can be configured by the user. Due to these differences in the interpretation of “throttle”, we display our data in increments of 1V or 2V, which eliminates the variable of a depleting battery supply voltage that makes other companies’ data confusing. Please refer to our blog post about `Understanding Vertiq Thrust Data <https://www.vertiq.co/blog/understanding-vertiq-thrust-data>`_ for more information.
+
+#. Why is my drone vibrating/oscillating excessively after taking off, causing it to be unstable?
```

Comment:
Why is my drone vibrating/oscillating excessively, causing instability, after take-off?